app/pods/forgot-password/controller.js

```diff
@@ -0,0 +1,32 @@
+import Controller from '@ember/controller';
+import { dropTask } from 'ember-concurrency-decorators';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class ForgotPasswordController extends Controller {
+  @service api;
+
+  @tracked isError = false;
```

Comment:
tasks have derived states for handling success and errors http://ember-concurrency.com/docs/derived-state